actions/get-vault-secrets/action.yaml

```
@@ -62,6 +62,14 @@ runs:
         COMMON_SECRETS: ${{ inputs.common_secrets }}
         REPO: ${{ github.repository }}
 
+    - id: get-github-jwt-token
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      with:
+        script: |
+          const jwt = await core.getIDToken();
+          core.setSecret(jwt);
+          core.setOutput("github_jwt",jwt);
```

super nit (and not required, I just think it looks nicer this way 😁)

```suggestion
          core.setOutput("github-jwt",jwt);
```